include/FileDialog.h

```diff
@@ -40,6 +42,8 @@ class LMMS_EXPORT FileDialog : public QFileDialog
 	explicit FileDialog( QWidget *parent = 0, const QString &caption = QString(),
 						const QString &directory = QString(),
 						const QString &filter = QString() );
+						
+	~FileDialog();
```

Comment:
We prefer to use the `override` keyword whenever overriding a virtual function (including a destructor), in accordance with the [Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c128-virtual-functions-should-specify-exactly-one-of-virtual-override-or-final).
```suggestion
	~FileDialog() override;
```